install/rbac/role.yaml

```diff
@@ -41,3 +121,69 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - operator.authorino.kuadrant.io
+  resources:
+  - authorinos
```

Comment:
I'm pushing a commit. IDK how this ended up here. This particular permission belongs to Authorino Operator, not to Authorino.